virt/virt-web-console-overview.adoc

```diff
@@ -774,7 +774,7 @@ You can manage disks on the *Disks* tab.
 |*Disks* table
 |List of VirtualMachine disks
 
-Click the Options menu {kebab} beside a disk to select *Edit* or *Detach*.
+Click the Options menu {kebab} beside a disk to select *Edit*, *Detach*, or *Make persistent*.
```

Comment:
```suggestion
Click the *Options* menu {kebab} beside a disk to select *Edit*, *Detach*, or *Make persistent*.
```

Reply:
From the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements): "Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable."